feature-libs/checkout/b2b/occ/config/default-occ-checkout-b2b-config.ts

```diff
@@ -15,9 +15,7 @@ export const defaultOccCheckoutB2BConfig: OccConfig = {
         setDeliveryAddress:
           'orgUsers/${userId}/carts/${cartId}/addresses/delivery',
         paymentTypes: 'paymenttypes',
-        // TODO:#checkout - shouldn't the following endpoints use `orgUsers`?
         setCartCostCenter: 'users/${userId}/carts/${cartId}/costcenter',
-        // TODO:#checkout - rename to b2bPlaceOrder? Similar to `scheduleReplenishmentOrder` from `feature-libs/checkout/scheduled-replenishment/occ/config/default-occ-scheduled-replenishment-config.ts`
```

Comment:
Reason why I didn't change it to b2bPlaceOrder is because we just changed the 'channel' (user vs orgUsers). Going by this logic, I would change `setDeliveryAddress` to `setB2bDeliveryAddress` and `getCheckoutDetails` to `getB2bCheckoutDetails`

Reply:
I'm willing to change the key to contain b2b, but let me know otherwise what you think if it's just for semantics. However, as mentioned in the comment about `scheduledReplenishmentOrder` key, I would say it wasn't named `placeOrder` because we're not placing an order, but scheduling a replenishment order :P. Basically, when we place an order, the order is placed, but when we scheduled one, it is scheduled to place an order at the given date/time.

Reply:
> Reason why I didn't change it to b2bPlaceOrder is because we just changed the 'channel' (user vs orgUsers). Going by this logic, I would change setDeliveryAddress to setB2bDeliveryAddress and getCheckoutDetails to getB2bCheckoutDetails

Yep, I don't mind doing this. I left a todo, and let's keep it in mind. If we decide to do it, let's do it at some point _after_ the cart merge, as we are making some changes to occ configs.